routes/api/stickers.js

```diff
@@ -103,14 +104,33 @@ router.post('/', isAdmin, imgController.uploadImg, controllerSticker.uploadFileS
 router.put('/:playerId', isAdmin, imgController.uploadImg, controllerSticker.uploadUpdatedFileSticker);
 
 //endpoint para borrar cromos
-router.delete('/:playerId', isAdmin, async (req,res)=>{
-    await Sticker.destroy({
-        where:{ id: req.params.playerId }
+router.delete('/:stickerId', isAdmin, async (req,res)=>{
+    const inInventory = await Inventory.findOne({
+      raw:true,
+      where: {stickerId : req.params.stickerId}
     });
-    res.json({ 
+    if (inInventory) {
+      return responses.errorDTOResponse(res, 400, "No se puede eliminar el cromo porque está en uso");
+    }
+    const inWarehouse = await Warehouse.findOne({
+      raw:true,
+      where: {stickerId : req.params.stickerId}
+    });
+    if (inWarehouse) {
+      return responses.errorDTOResponse(res, 400, "No se puede eliminar el cromo porque está en uso");
+    }
+    try{
+      await Sticker.destroy({
+        where:{ id: req.params.stickerId }
+    });
+    return res.status(200).json({ 
       success:true, 
       message:"item deleted"
```

Comment:
```suggestion
    return responses.successDTOResponse(res, 200, "Sticker eliminado con exito");
```
OJO PELAO, NO USA RESPONSES
Revisen la sugerencia igual, que furule xd